src/work-allocation-2/containers/available-tasks/available-tasks.component.ts

```diff
@@ -0,0 +1,129 @@
+import { Component } from '@angular/core';
+import { InfoMessage, InfoMessageType, TaskActionIds } from '../../../work-allocation-2/enums';
+import { Location, SearchTaskRequest } from '../../../work-allocation-2/models/dtos';
+import { InvokedTaskAction, Task } from '../../..//work-allocation-2/models/tasks';
+import { handleFatalErrors, REDIRECTS } from '../../../work-allocation-2/utils';
+import { TaskFieldConfig } from '../../../work-allocation/models/tasks';
+
+
+import { ConfigConstants, ListConstants, SortConstants } from '../../components/constants';
+import { TaskListWrapperComponent } from '../task-list-wrapper/task-list-wrapper.component';
+
+@Component({
+  selector: 'exui-available-tasks',
+  templateUrl: 'available-tasks.component.html'
+})
+export class AvailableTasksComponent extends TaskListWrapperComponent {
+  private selectedLocations: Location[];
+
+  public get fields(): TaskFieldConfig[] {
+    return ConfigConstants.AvailableTasks;
+  }
+
+  public get sortSessionKey(): string {
+    return SortConstants.Session.AvailableTasks;
+  }
+
+  public get view(): string {
+    return ListConstants.View.AvailableTasks;
+  }
+
+  /**
+   * Override the default.
+   */
+  public getSearchTaskRequest(): SearchTaskRequest {
+    return {
+      search_parameters: [
+        this.getLocationParameter(),
+        { key: 'state', operator: 'IN', values: ['unassigned'] }
+      ],
+      sorting_parameters: [this.getSortParameter()]
+    };
+  }
+
+  /**
+   * When the filter changes, we need to reload the list of tasks.
+   * @param locations The currently selected locations.
+   */
+  public onLocationsChanged(locations: Location[]): void {
+    this.infoMessageCommService.removeAllMessages();
+    this.selectedLocations = [ ...locations ];
+    this.loadTasks();
```

Comment:
This won't work because there isn't currently a filter on the page - what I have done is put this in a comment with the relevant EUI number stating why this will be necessary in the future. If you agree with this please can you resolve convo? If not let me know.

Reply:
In which case, is this entire method redundant, along with `getLocationParameter()`, and the `selectedLocations` property?

Reply:
Changes made to getLocationParameter, etc as mentioned